src/test/rustdoc-ui/check-doc-attrs.stderr

```diff
@@ -0,0 +1,52 @@
+warning: this attribute can only be applied to a `use` item
+  --> $DIR/check-doc-attrs.rs:1:8
+   |
+LL | #![doc(inline)]
+   |        ^^^^^^
+   |
+   = note: read https://doc.rust-lang.org/nightly/rustdoc/the-doc-attribute.html#docno_inlinedocinline for more information
```

Comment:
It would be nice to point to the item here:
```
warning: this attribute can only be applied to a `use` item
  --> $DIR/check-doc-attrs.rs:1:8
   |
LL | #![doc(inline)]
   |        ^^^^^^
   | struct Foo;
   | ^^^^^^^^^^^ not a `use` item
   = note: read https://doc.rust-lang.org/nightly/rustdoc/the-doc-attribute.html#docno_inlinedocinline for more information